roles/maas_postgres/tasks/teardown.yaml

```diff
@@ -7,35 +7,25 @@
   ansible.builtin.set_fact:
     maas_postgres_config_dir: "/etc/postgresql/{{ maas_postgres_version_number }}/main/"
 
-- name: "Uninstall libpq-dev for Configuration"
+- name: "Uninstall Postgres"
   ansible.builtin.apt:
-    name: libpq-dev
-    update_cache: true # update cache still if pip was already installed
+    name: "{{ maas_postgres_deb_name | default('postgresql-{{ maas_postgres_version_number }}') }}"
     state: absent
+    purge: true
 
-- name: "Uninstall psycopg2 For Configuration"
-  ansible.builtin.pip:
-    name: "psycopg2"
-    version: "2.9.3"
+- name: "Uninstall libpq-dev"
+  ansible.builtin.apt:
+    name: libpq-dev
     state: absent
 
-- name: "Uninstall acl to become postgres user"
+- name: "Uninstall acl"
   ansible.builtin.apt:
     name: acl
     state: absent
 
-- name: "Uninstall Postgres"
-  ansible.builtin.apt:
-    name: "{{ maas_postgres_deb_name | default('postgresql-{{ maas_postgres_version_number') }}"
-    update_cache: true # update cache still if previous packages are already installed
+- name: "Uninstall psycopg2"
+  ansible.builtin.pip:
+    name: "psycopg2"
+    version: "2.9.3"
```

Comment:
why hardcode the version?

Reply:
I think the install task that I copied had this hardcoded, but yeah that shouldn't be needed for teardown